airbyte/sources/util.py

```
@@ -45,6 +45,7 @@ def get_connector(
 def get_source(  # noqa: PLR0913 # Too many arguments
     name: str,
     config: dict[str, Any] | None = None,
+    config_change_callback: Callable[[dict[str, Any], int], None] | None = None,
```

Ruff

Undefined name `Callable`

(F821)